cartography/intel/azure/sql.py

```diff
@@ -0,0 +1,735 @@
+import logging
```

Comment:
code looks good now. Please change the entity names as i have highlighted in indexes.cypher file.

Reply:
done